### DISCUSSION TITLE
[Feature Request] They could add a new component, something like a carousel with navigation

### DISCUSSION THREAD

**Discussion by royaltics-solutions** [heroui-inc/heroui#1358] [Feature requests]
### Is your feature request related to a problem? Please describe.

Hello, thank you for the effort you give to provide us with this library.
I am a web page developer, mostly for business page design and portfolios.
I use your library a lot, in many cases I have come across the need to add a carousel of images or content in the pages and start and I depend on other libraries, but I would like your library to include this plugin in the future.
**Could you please add something like a carousel with navigation**

### Describe the solution you'd like

As a designer, I would like a component that is not so complex, but that allows the essentials and with default animations.
1. That allows reading an Array of Objects that at least allows us to define, title, description and background image or background color.
2. That allows you to choose between automatic tour for some time interval or with navigation buttons

### Describe alternatives you've considered

I currently use Swiper Element (WebComponent).
But frankly I think it can be improved

**Comment by wingkwong** [comment]
let's track in #4678
